pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/startree/StarTreeQueryGenerator.java

```diff
@@ -82,6 +83,16 @@ private String generateAggregation(String metricColumn) {
         metricColumn);
   }
 
+  private String generateFilteredAggregation(String metricColumn) {
```

Comment:
@Jackie-Jiang these tests seem to consistently fail when run using the multi-stage engine.  Are filtered aggregations supported in multi-stage, or not yet?

Reply:
attempted fix in https://github.com/apache/pinot/pull/11886/commits/990bf5a725ef50c256f228e9415eea49566870fe